webrender/src/clip_scroll_node.rs

```diff
@@ -37,7 +37,7 @@ pub struct ClipInfo {
     /// The final transformed rectangle of this clipping region for this node,
```

Comment:
I think it's useful to say "the bounding rectangle of the transformed rectangle" in the comment just to clarify. Might also be worth mentioning the other part of the tuple as well.